src/main/scala/sangria/macros/derive/DeriveObjectTypeMacro.scala

```diff
@@ -357,4 +366,5 @@ class DeriveObjectTypeMacro(context: blackbox.Context) extends {
   case class MacroExcludeFields(fieldNames: Set[String], pos: Position) extends MacroDeriveObjectSetting
   case class MacroAddFields(fields: List[Tree]) extends MacroDeriveObjectSetting
   case class MacroReplaceField(fieldName: String, field: Tree, pos: Position) extends MacroDeriveObjectSetting
+  case class MacroTransformFieldNames(transformerName: Symbol) extends MacroDeriveObjectSetting
```

Comment:
Using a `Symbol` here concerns me a bit. Maybe it would be better to define it as `transformer: Tree`? WDYT?